blockchain/blockindex.go

```diff
@@ -319,6 +319,44 @@ func (bi *blockIndex) UnsetStatusFlags(node *blockNode, flags blockStatus) {
 	bi.Unlock()
 }
 
+// InactiveTips returns all the block nodes that aren't in the best chain.
+//
+// This function is safe for concurrent access.
+func (bi *blockIndex) InactiveTips(bestChain *chainView) []*blockNode {
+	bi.RLock()
+	defer bi.RUnlock()
+
+	// Look through the entire blockindex and look for nodes that aren't in
+	// the best chain. We're gonna keep track of all the orphans and the parents
+	// of the orphans.
+	orphans := map[chainhash.Hash]*blockNode{}
```

Comment:
Sort of non-traditional syntax here? Did a double take as was expecting:
```go
orphans := make(map[chainhash.Hash]*blockNode)
```

Usually this syntax is used to create a map literal where you already know some/all of the values.